docs/mp/guides/04_health.adoc

```diff
@@ -529,14 +528,14 @@ kubectl delete -f ./health.yaml
 ----
 
 === Summary
-This guide demonstrated how to use health-check in a Helidon MP application as follows:
+This guide demonstrated how to use health checks in a Helidon MP application as follows:
 
-* Access the default health-check
+* Access the default health check
```

Comment:
Plural here? There are multiple built-in health checks.

Also, earlier we called them "built-in." Here, "default."  Maybe use "built-in" here for consistency.